LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2017 Portal Architects, Inc.
+Copyright (c) 2017 SkySync, Inc.
```

Comment:
Want to up the year?

Reply:
should i?  i thought the copyright year is supposed to be when it was first made available...